packages/encoding/test/decodeAssetId.test.ts

```diff
@@ -1,6 +1,6 @@
 import { expect } from 'chai'
 
-import { decodeAssetId, encodeAssetId } from '../src/assetId'
+import { decodeAssetId, encodeAssetId } from '../src/decodeAssetId'
```

Comment:
nit: I'm not very attached to file names personally, but `encodeAssetId` is now the living in the shadow of its inverse function :P

Reply:
Oh you're right. I'm going to change the file name